### PR TITLE
Created resistance experiment

### PR DESCRIPTION
**Pull request by abhinavraj23**
Created resistance experiment according to new template

### CODE REVIEW

File: contribute/basic/Resistance_measurement.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Connect the two resistors in series as shown below
+
+<img src="https://fossasia.github.io/pslab-experiments/images/schematics/RSeriesSimple.svg" width=300>
+
+#### In Parallel
+
+Parallel connection is to be made in this manner
+
+<img src="https://fossasia.github.io/pslab-experiments/images/schematics/RParallelSimple.svg" width=300>
```
Comment (CloudyPadmal) [inline contribute/basic/Resistance_measurement.md]: One image has big labels compared to the other. Can we make them uniform?

File: contribute/basic/Resistance_measurement.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+#### For Series
+
+You will observe that the total resistance measured when combined will be the sum of the individual resistances.
+
+Thus we conclude that in case of series resistance Rtotal = R1+R2
+#### For Parallel
+
+You will observe that the reciprocal of the total resistance measured when combined in parallel will be the sum of the reciprocal of individual resistances
+
+Thus we conclude that in case of parallel resistance
```
Comment (CloudyPadmal) [inline contribute/basic/Resistance_measurement.md]: This has come inline

**Comment by abhinavraj23** [comment]
@CloudyPadmal Changes done :+1:

**Comment by cweitat** [comment]
Breadboard design is not included, components table is not followed.
instead of multiple resistors, please specify exactly which resistors.
Don't add you own section. Please follow the template. Is expected results and not observation

**Comment by cweitat** [comment]
https://github.com/fossasia/pslab-experiments/blob/master/contribute/README.md

Read the readme if unsure

**Pull request closed without merging**